packages/shell-chrome/views/_partials/footer.edge

```diff
@@ -4,11 +4,13 @@
 >
     <div class="flex-1">
         <div class="flex items-center text-xs leading-9 font-medium font-mono">
-            <div 
-                class="flex-1 pl-3" 
-                x-text="footerText"
-                data-testid="footer-line"
-            ></div>
+            <div class="flex-1 pl-3" data-testid="footer-line">
+                Watching
+                <a href="#" data-testid="footer-components-link" @click.prevent="activeTab = 'components'"><span x-text="components.length"></span> component(s)</a>{{ devOnly ? ', ' : '' }}
+                @if(devOnly)
```

Comment:
Is this supposed to be wrapped in a `devOnly` check?

Reply:
Yeah, I'll remove the devOnly checking for the warnings tab when it's ready to go